schemas/entity/task.schema.tpl.json

```diff
@@ -0,0 +1,32 @@
+{
+  "comment": "Do we really need this or just a protocol is enough?",
+  "_type": "https://openminds.ebrains.eu/MRI/Task",
```

Comment:
I do not get what this is about... there should not be any other protocol (or behavioral protocol).